packages/adminrouter/docker/adminrouter-upstreams-ee.conf

```diff
@@ -1,17 +1,14 @@
 set $upstream_networking_api http://127.0.0.1:61430;
 
-proxy_ssl_trusted_certificate /run/dcos/pki/CA/certs/ca.crt;
+proxy_ssl_trusted_certificate /run/dcos/pki/CA/ca-bundle.crt;
 proxy_ssl_verify on;
 proxy_ssl_verify_depth 5;
 
-# For the sake of tests simplicity, we will not be using/testing leader.mesos
-# alias here. This should be done by integration tests.
-set $upstream_mesos http://127.0.0.2:5050;
-set $upstream_marathon http://127.0.0.1:8080;
+set $upstream_mesos http://leader.mesos:5050;
```

Comment:
My concern with this patch is whether we can make things worse in respect of existing `leader.mesos` TTL.  We had 30 seconds refresh interval that forced re-resolving the leader. Our DNS service has set TTL to 60 seconds so we can possibly double the time for new leader detection.